whatshap/cli/haplotagphase.py

```diff
@@ -39,6 +40,7 @@ def add_arguments(parser):
     arg("--chromosome", dest="chromosomes", metavar="CHROMOSOME", default=[], action="append",
         help="Name of chromosome to phase. If not given, all chromosomes in the input VCF are phased. "
         "Can be used multiple times.")
+    arg("--no-mav", dest="ignore_mav", default=False, action="store_true", help="Ignore multiallelic variants.")
```

Comment:
To avoid the double negative of `ignore_mav=False`, please invert the logic:
```suggestion
    arg("--no-mav", dest="mav", default=True, action="store_false", help="Ignore multiallelic variants.")
```